src/ploomber/tasks/notebook.py

```diff
@@ -588,6 +600,8 @@ def __init__(
         dag,
         name=None,
         params=None,
+        ploomber_engine=False,
```

Comment:
the option should be `engine`, since that's more flexible in case we want to add new engines down the road. should take `ploomber-engine`, `papermill` as options. with papermill being the default. we should also validate the values and throw a ValueError if another value is passed